demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -776,9 +780,21 @@ def test_modeling_rule(
     Test a modeling rule against an XSIAM tenant
     """
     logging_setup(
-        console_log_threshold=console_log_threshold,
-        file_log_threshold=file_log_threshold,
-        log_file_path=log_file_path,
+        console_log_threshold=get_config_param_kwarg_env(
+            console_log_threshold,
+            "DEMISTO_SDK_CONSOLE_LOG_THRESHOLD",
+            logging.INFO,
+        ),
+        file_log_threshold=get_config_param_kwarg_env(
+            file_log_threshold,
+            "DEMISTO_SDK_FILE_LOG_THRESHOLD",
+            logging.DEBUG,
```

Comment:
Can we use `logging.DEBUG` from the imported logger, instead of importing the `logging` just for this constant?